tox.ini

```diff
@@ -25,8 +25,9 @@ commands = pytest -vvs {toxinidir}/tests/unit {posargs}
 
 [testenv:integration]
 deps =
+    {[testenv]deps}
     pytest-operator
-commands = pytest -vvs --log-cli-level=INFO {toxinidir}/tests/integration {posargs}
+commands = pytest -vvs --tb native --log-cli-level=INFO --asyncio-mode=auto {toxinidir}/tests/integration {posargs}
```

Comment:
Why have these arguments been added?

Reply:
`--asyncio-mode` is added to eliminate warnings coming from pytest_asyncio
```
DeprecationWarning: '@pytest.fixture' is applied to <fixture abort_on_fail, file=/home/natasha/WorkRepo/dex-auth-operator/.tox/integration/lib/python3.8/site-packages/pytest_operator/plugin.py, line=135> in 'legacy' mode, please replace it with '@pytest_asyncio.fixture' as a preparation for switching to 'strict' mode (or use 'auto' mode to seamlessly handle all these fixtures as asyncio-driven).
```

`--tb native` is for using python standard library formatting in tracebacks. I see this flag set in oidc-gatekeeper and operator-template. I added it in for consistency sake